algebird-spark/src/main/scala/com/twitter/algebird/spark/implicits/package.scala

```diff
@@ -0,0 +1,29 @@
+package com.twitter.algebird.spark
+
+package object implicits {
+
+import com.twitter.algebird.BF
+
+import com.twitter.algebird.BFZero
+
+import java.util.PriorityQueue
+
+import com.twitter.algebird.BloomFilterMonoid
+
+import org.apache.spark.sql.Encoder
+
+  import scala.reflect.ClassTag
+  implicit def kryoPriorityQueueEncoder[A](implicit ct: ClassTag[PriorityQueue[A]]): Encoder[PriorityQueue[A]] =
+    org.apache.spark.sql.Encoders.kryo[PriorityQueue[A]](ct)
```

Comment:
```suggestion
    org.apache.spark.sql.Encoders.kryo[PriorityQueue[A]]
```
iirc `ClassTag` is a context bound so the above should be possible?